gulpfile.js

```diff
@@ -47,10 +42,10 @@ const TEMPLATE_DIRECTORY = 'templates';
 ////////////////////////////////////////////////////////////////////////////////
 // WEBPACK
 ////////////////////////////////////////////////////////////////////////////////
-var webpackConfig = 
+var webpackConfig =
```

Comment:
Try to avoid whitespace commits, or if you must place them all together in a single commit.

Reply:
Sorry about that!